[CI] SmokeTestIngestWithAllDepsClientYamlTestSuiteIT test {yaml=ingest/100_sampling_with_reroute/Test get sample with multiple reroutes} failing

**Build Scans:**
- [elasticsearch-periodic #11391 / release-tests](https://gradle-enterprise.elastic.co/s/uu4ahos24mf6w)
- [elasticsearch-periodic-java-ea #70 / release-tests](https://gradle-enterprise.elastic.co/s/clzo3ucg7ki2o)
- [elasticsearch-pull-request #101923 / checkpart6_release-tests](https://gradle-enterprise.elastic.co/s/vzhbdxn65pqds)
- [elasticsearch-pull-request #101866 / checkpart6_release-tests](https://gradle-enterprise.elastic.co/s/wgnesizsfl6da)
- [elasticsearch-periodic #11387 / release-tests](https://gradle-enterprise.elastic.co/s/qiywaixb6ko3c)
- [elasticsearch-periodic-java-ea #69 / release-tests](https://gradle-enterprise.elastic.co/s/67ydv6hdblqji)

**Reproduction Line:**
```
./gradlew ":qa:smoke-test-ingest-with-all-dependencies:yamlRestTest" --tests "org.elasticsearch.smoketest.SmokeTestIngestWithAllDepsClientYamlTestSuiteIT.test {yaml=ingest/100_sampling_with_reroute/Test get sample with multiple reroutes}" -Dtests.seed=B1BFFB38FE2CC705 -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dlicense.key=x-pack/license-tools/src/test/resources/public.key -Dtests.locale=syr-SY -Dtests.timezone=Europe/Amsterdam -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.smoketest.SmokeTestIngestWithAllDepsClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Byaml%3Dingest%2F100_sampling_with_reroute%2FTest%20get%20sample%20with%20multiple%20reroutes%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [ingest/100_sampling_with_reroute:123]: expected [2xx] status code but api [indices.put_sample_configuration] returned [400 Bad Request] [{"error":"no handler found for uri [/foo/_sample/config?error_trace=true] and method [PUT]"}]
```

**Issue Reasons:**
- [main] 4 consecutive failures in step release-tests
- [main] 2 consecutive failures in step checkpart6_release-tests
- [main] 6 failures in test test {yaml=ingest/100_sampling_with_reroute/Test get sample with multiple reroutes} (4.6% fail rate in 130 executions)
- [main] 4 failures in step release-tests (100.0% fail rate in 4 executions)
- [main] 2 failures in step checkpart6_release-tests (100.0% fail rate in 2 executions)
- [main] 2 failures in pipeline elasticsearch-periodic (100.0% fail rate in 2 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (2.8% fail rate in 71 executions)
- [main] 2 failures in pipeline elasticsearch-periodic-java-ea (100.0% fail rate in 2 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -159,9 +159,6 @@ tests:
 - class: org.elasticsearch.xpack.inference.DefaultEndPointsIT
   method: testMultipleInferencesTriggeringDownloadAndDeploy
   issue: https://github.com/elastic/elasticsearch/issues/117208
-- class: org.elasticsearch.smoketest.SmokeTestIngestWithAllDepsClientYamlTestSuiteIT
-  method: test {yaml=ingest/100_sampling_with_reroute/Test get sample with multiple reroutes}
-  issue: https://github.com/elastic/elasticsearch/issues/137457
 - class: org.elasticsearch.xpack.ilm.CCRIndexLifecycleIT
   method: testTsdbLeaderIndexRolloverAndSyncAfterWaitUntilEndTime {targetCluster=FOLLOWER}
   issue: https://github.com/elastic/elasticsearch/issues/137565

